clusters/falco.yaml

```diff
@@ -0,0 +1,31 @@
+# The Falco Authors
```

Comment:
We changed the repo structure to the following:
```
tree clusters
clusters
├── projects
│   └── falco
│       └── falco-namespace.yaml
├── base
│   ├── kepler-grafana.yaml
│   ├── kepler.yaml
│   └── kube-prometheus-stack.yaml
└── flux-system
    ├── gotk-components.yaml
    ├── gotk-sync.yaml
    └── kustomization.yaml
```
 Could move the config to `clusters/projects/falco` please?